erts/emulator/beam/jit/arm/32/instr_bif.cpp

```diff
@@ -0,0 +1,278 @@
+/*
+ * %CopyrightBegin%
+ *
+ * Copyright Ericsson AB 2020-2023. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * %CopyrightEnd%
+ */
+
+#include "beam_asm.hpp"
+
+extern "C"
+{
+#include "beam_common.h"
+#include "code_ix.h"
+#include "erl_bif_table.h"
+#include "erl_nfunc_sched.h"
+#include "bif.h"
+#include "erl_msacc.h"
+}
+
+void BeamModuleAssembler::ubif_comment(const ArgWord &Bif) {
+    // TODO
+    ASSERT(false);
```

Comment:
`emit_nyi(...)` is a better way to do this: you crash when encountering the unimplemented instruction (or unimplemented _part_ of the instruction).

From experience, having to do everything up front increases the risk of bugs sneaking in.

Reply:
Thank you very much for the comments. Using `emit_nyi(...)` sounds like a good idea. We weren't aware of that.